modules/etcd-creating-automated-backups.adoc

```diff
@@ -0,0 +1,355 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc
+
+:_content-type: PROCEDURE
+[id="creating-automated-etcd-backups_{context}"]
+= Creating automated etcd backups
+
+The automated backup feature for etcd supports both recurring and single backups. Recurring backups create a cron job that starts a single backup each time it triggers.
+
+:FeatureName: Automating etcd backups
+include::snippets/technology-preview.adoc[]
+
+[id="enabling-automated-etcd-backups_{context}"]
+== Enabling automated etcd backups
+
+Follow these steps to enable the automated backups feature for etcd.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `FeatureGate` custom resource (CR) file named `etcd-enable-automated-backups.yaml` with the following contents:
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
+====
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade
+----
+
+. It takes time to enable the related APIs before you can apply the CR. Verify the creation of the custom resource definition (CRD) before applying the CR by running the following command:
++
+[source,terminal]
+----
+$ oc get crd | grep backup
+----
++
+.Example output
+[source,terminal]
+----
+backups.config.openshift.io 2023-10-25T13:32:43Z
+etcdbackups.operator.openshift.io 2023-10-25T13:32:04Z
+----
+
+. Apply the CR and enable automated backups:
++
+[source,terminal]
+----
+$ oc apply -f etcd-enable-automated-backups.yaml
+----
+
+[id="creating-single-etcd-backup_{context}"]
+== Creating a single etcd backup
+
+Follow these steps to create a single etcd backup by creating and applying a custom resource (CR).
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a CR file named `etcd-single-backup.yaml` with contents such as the following example:
```

Comment:
If by "such as", you mean "similar to", you can replace "such as" with "like".

Reply:
Oh, interesting! The preference for "such as" must be a Red Hat thing, then. I have to confess that I know IBM Style best since it was my sole guidebook for so many years, and it recommends using "like" when you mean "similar to" and "such as" otherwise. Feel free to use "such as" if Vale doesn't like "like".